doc/Introduction.html

```diff
@@ -69,30 +69,31 @@ <h2 class="sidebar-projectVersion">
       
         <section id="moduledoc" class="docstring">
           <h1>Welcome to the Airport</h1>
-<p>You made it ! Let’s go through some information before you get on the plane.</p>
+<p>You made it! Let’s go through some information before you get on the plane.</p>
 <p>On each destination you’ll receive information of your next trip.</p>
 <p>Here are suggestions for your safe and successful travels.</p>
 <ul>
-<li>Remember that you are an engineer and you are here to create solutions.
+<li>Remember that you are a great student and you are here to create solutions.
```

Comment:
Yeah ! It makes more sense. We want them to feel comfortable and this helps a lot. :shipit:

- Alan

> On Nov 29, 2015, at 12:09 PM, Ana Castro <notifications@github.com> wrote:
> 
> In doc/Introduction.html:
> 
> >  <p>On each destination you’ll receive information of your next trip.</p>
> >  <p>Here are suggestions for your safe and successful travels.</p>
> >  <ul>
> > -<li>Remember that you are an engineer and you are here to create solutions.
> > +<li>Remember that you are a great student and you are here to create solutions.
> @alan-andrade what do you think about using "students" instead of "engineers"? I think this way we can make people feel more identified and comfortable with their current role in the workshop. 
> 
> —
> Reply to this email directly or view it on GitHub.
>